partcad/src/partcad/part_factory_scad.py

```
@@ -74,16 +81,16 @@
                 try:
-                    # First, make sure it's not the known problem in Mesher
-                    shape = b3d.import_stl(stl_path).wrapped
-                except Exception as e:
-                    part.error("%s: %s" % (part.name, e))
-                    return None
+                    shape = b3d.Mesher().read(stl_path)[0].wrapped
+                except:
+                    try:
+                        # First, make sure it's not the known problem in Mesher
+                        shape = b3d.import_stl(stl_path).wrapped
+                    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)